src/client.rs

```diff
@@ -531,7 +531,7 @@ where
                     error_response(
                         &mut write,
                         &format!(
-                            "No pool configured for database: {:?}, user: {:?}",
+                            "No pool configured for database: {:?}, user: {:?} (in startup)",
```

Comment:
This is debug level messaging we should not be returning to the client.

Reply:
Good point. Reverted this.